### DISCUSSION TITLE
Tags for search options

### DISCUSSION THREAD

**Discussion by lgalloherran** [twilio-labs/paste#1005] [Q&A] [answered]
Hi Team! I'm working on the number provision search filter, and I'm trying to reduce the size of the 'advance search options' after hitting on 'search results. That's why I'm wondering if there exists tags or an equivalent component in Paste that I can use. Or if there's a better way so that the 'Advance search' will not take the whole screen. 
This the how it looks when the 'Advance search' is expanded: 
![Captura de Pantalla 2020-12-07 a la(s) 6 55 28 p  m](https://user-images.githubusercontent.com/72419222/101419334-c8934080-38bd-11eb-856d-85085d9d3d5f.png)

**Comment by serifluous** [comment]
Hey @lgalloherran ! Can you give us more context about which part of this screen is the advanced search filters vs. which are just the regular filters? And what do you need the tags to do (example: whether they need to be interactive/focusable, what filters they're supposed to replace, etc)?

The short answer is that we don't have tags or badges currently, but they are on our roadmap. We can potentially point you to a few product designers who have been doing similar work on filters and styling badges/tags using [Box](https://paste.twilio.design/primitives/box/), [Text](https://paste.twilio.design/primitives/text/), and [tokens](https://paste.twilio.design/tokens/) in the meantime.

**Comment by lgalloherran** [comment]
Hi Sarah, 
This will be the 'light' version of the search: 
![Captura de Pantalla 2020-12-09 a la(s) 10 38 53 a  m](https://user-images.githubusercontent.com/72419222/101651166-becb2380-3a0a-11eb-8f40-5214d81f5fd6.png)

I'm hoping of using the tags in order to reduce the space that other search elements take.

**Reply by lgalloherran** [reply]
@serifluous I have done a version with tags: (So once you hit search, the 'advanced search' section collapses and shows the tags: 
![Captura de Pantalla 2020-12-09 a la(s) 5 29 52 p  m](https://user-images.githubusercontent.com/72419222/101696548-46815400-3a44-11eb-9401-f83b5420c36f.png)

**Comment by serifluous** [comment, marked as answer]
Hey @lgalloherran !

For advanced search, using the Disclosure to collapse the additional form fields like you have here works. However, you won't be able to add any focusable element (like dismissible tags) in the Disclosure heading. Take a look at our [accessibility guidelines around Disclosures](https://paste.twilio.design/components/disclosure/#accessibility).

You can check out how [Messaging Insights displays their filters when the Disclosure is collapsed](https://www.twilio.com/console/sms/insights/delivery) and reach out to the Messaging designers to chat about whether using a similar pattern might be appropriate for your case.
